downloads/swan-lake-release-notes/swan-lake-preview5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of Ballerina Swan Lake Preview 5 
+
+This release is the fifth preview version of Ballerina Swan Lake. This release includes a new set of language features along with improvements and bug fixes to the compiler, runtime, standard libraries, and developer tooling.
+
+- [Updating Ballerina](#updating-ballerina)
+    - [For existing users](#for-existing-users)
+    - [For new users](#for-new-users)
+- [What is new in Ballerina Swan Lake Preview 5](#what-is-new-in-ballerina-swan-lake-preview-5)
+    - [Language](#language)
+        - [Isolated objects](#isolated-objects)
+        - [Removed XML functional constructors](#removed-xml-functional-constructors)
+        - [Unified object method scope and object field scope](#unified-object-method-scope-and-object-field-scope)
+        - [Improved lang library functions](#improved-lang-library-functions)
+    - [Runtime](#runtime)
+    - [Standard Library](#standard-library)
+        - [HTTP client remote function API improvement with data-binding support](#http-client-remote-function-api-improvement-with-data-binding-support)
+        - [SQL procedure call API improvement](#sql-procedure-call-api-improvement)
+        - [Add misfire policy](#add-misfire-policy)
+    - [Developer Tools](#developer-tools)
+        - [Code formatting](#code-formatting)
+
+### Updating Ballerina
+
+You can use the update tool to update to Ballerina Swan Lake Preview 5 as follows.
+
+#### For existing users
+
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](http://ballerina.io/swan-lake/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
+                        
+> `ballerina update`
+
+ Next, execute the command below to update to Swan Lake Preview 4.
+
+ > `ballerina dist pull slp5b1`                 
```

Comment:
Shouldn't this be `ballerina dist pull slp5` ? @keizer619

Reply:
yes slp5 above name is a temporary one for current build

Reply:
Fixed.